app/assets/stylesheets/application.scss

```diff
@@ -2,4 +2,36 @@
 @import "lib/hmrc-frontend/hmrc/all";
 
 $hmrc-assets-path: "/minor-entity-identification-frontend/assets/lib/hmrc-frontend/hmrc/";
-$govuk-assets-path: "/minor-entity-identification-frontend/assets/lib/govuk-frontend/govuk/assets/";
\ No newline at end of file
+$govuk-assets-path: "/minor-entity-identification-frontend/assets/lib/govuk-frontend/govuk/assets/";
+
+.app-reference-number {
+  display: block;
+  font-weight: bold;
+}
+
+.autocomplete__menu {
+  font-family: "GDS Transport", Arial, sans-serif;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+  font-weight: 400;
+  font-size: 16px;
+  font-size: 1rem;
+  line-height: 1.25;
+  color: #0b0c0c;
+}
+
+.app-reference-number {
+  display: block;
+  font-weight: bold;
+}
+
+.autocomplete__menu {
+  font-family: "GDS Transport", Arial, sans-serif;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+  font-weight: 400;
+  font-size: 16px;
+  font-size: 1rem;
+  line-height: 1.25;
+  color: #0b0c0c;
+}
```

Comment:
duplicates of above. As described in slack to get both the footer working and the font working for the autocomplete suggestion dropdown I think the  .autocomplete__menu section needs be moved to accessible-autocomplete-2.0.2.min.css and then it coild be removed from here